probe-rs/src/bin/probe-rs/cmd/dap_server/debug_adapter/dap/request_helpers.rs

```diff
@@ -334,7 +334,7 @@ pub(crate) fn halt_core(
 ) -> Result<probe_rs::CoreInformation, DebuggerError> {
     match target_core.halt(Duration::from_millis(100)) {
         Ok(cpu_info) => Ok(cpu_info),
-        Err(error) => Err(DebuggerError::Other(anyhow!("{}", error))),
+        Err(error) => Err(DebuggerError::Other(anyhow!("{:#}", error))),
```

Comment:
```suggestion
        Err(error) => Err(DebuggerError::Other(anyhow!("{error:#}"))),
```

Just a suggestion to migrate to the new syntax when we change existing messages.

Reply:
I think we can just use `anyhow!(error)` here, I don't think converting it to a string has any benefits?